### DISCUSSION TITLE
Permitted development rights

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#39] [Planning consideration]
# Permitted development rights

## What we've done

We've created the main [permitted development right schema](https://raw.githubusercontent.com/digital-land/specification/main/content/dataset/permitted-development-right.md).

We also needed to create a category-type dataset called [permitted development right part](https://raw.githubusercontent.com/digital-land/specification/main/content/dataset/permitted-development-right-part.md).

The [Permitted development right data](https://www.planning.data.gov.uk/dataset/permitted-development-right) we created as a starting point is available on planning.data.gov.uk.

## What we're looking for

We'd like to know:

* does the data make sense to you?
* does it include everything you'd expect it to use?
* are you using the data? How are you using it?
* how would you improve the permitted development data?
* anything else you think we should know?

**Comment by colmjude** [comment]
We have extracted the permitted developments from the legislation and added them to the [`permitted-development-right`](https://www.planning.data.gov.uk/dataset/permitted-development-right) and [`permitted-development-right-part`](https://www.planning.data.gov.uk/dataset/permitted-development-right-part) datasets.

If you think we are missing any or something isn't right, please let us know so we can update it.

**Comment by trodaway** [comment]
With regards to the article 4 layer, and the permitted development rights being revoked, if our directions only revoke part of a clause, do we need to request the dataset be expanded to include these or do we just note that it's only relevant to one part?

For example, we have a [direction for Whalton](https://www.northumberland.gov.uk/NorthumberlandCountyCouncil/media/Planning-and-Building/Conservation/Whalton-Signed-Direction.pdf), based on the 1995 order, that talks about swimming pools; however the applicable bit on the 2015 order, as used by the [permitted-development-right](https://www.planning.data.gov.uk/dataset/permitted-development-right) dataset, includes lpg/oil storage containers.

[1E: Buildings etc incidental to the enjoyment of a dwellinghouse](https://www.planning.data.gov.uk/entity/11500005)
> The provision within the curtilage of the dwellinghouse of: (a) any building or enclosure, swimming or other pool required for a purpose incidental to the enjoyment of the dwellinghouse as such, or the maintenance, improvement or other alteration of such a building or enclosure; (b) a container used for domestic heating purposes for the storage of oil or liquid petroleum gas.

Alternatively, is there a nice guide of how these older orders map over to the most recent?

**Reply by kieranwint** [reply]
Hi @trodaway,

Thanks for raising this, and apologies for the delayed response.
Reading through the example you've provided, the recommendation would be to use 1E in it's entirety and add a note to that particular data row explaining that it's only a segment of it. Wherever there is supplementary information for data, we would recommend the notes field.

Where we have captured permitted development rights in the dataset that we maintain, [(for example, 1E)](https://www.planning.data.gov.uk/entity/11500005), we have captured both parts a) and b) in the same description, and similarly, where there are multiple sub-paragraphs in a given class, this has been captured in totality.

Just out of interest, would you say there would be numerous use cases for breaking this information out? Also, more generally, would there be anything further you might suggest that would improve this dataset?

Hopefully this helps, thanks in advance!
Kieran

**Reply by trodaway** [reply]
Thanks for the reply @kieranwint.

Okay, I'll do that - it's what I had been doing already.

> Just out of interest, would you say there would be numerous use cases for breaking this information out? 

I suppose having it broken out keeps the data within the schema such that it can be queried properly, whereas having the data about which part within free text doesn't allow this. I believe we have some where there's numerous rights being removed, and within that more than one is only a partial one, so it could keep it cleaner and prevent confusion.

>Also, more generally, would there be anything further you might suggest that would improve this dataset?

Having the historical rights on there too would be useful, or clearer guidance of how it links to the one used. I'm not a planning professional, rather in the GIS team, so perhaps there is guidance out there that I haven't found yet. This also makes it future proof - from what I can tell the 2015 legislation is used, so this might not align with the 2024 and any future legislation. If it's just updated to the future legislation in place, this could cause issues where parts have been quoted above that don't match up in the new legilsation (e.g. if a new part a was added, moving the old parts a and b to parts b and c respectively).

**Reply by kieranwint** [reply]
@trodaway, this is useful to know so thank you for bringing it to our attention. 
Regarding historical data, if there were particular historical permitted development rights you currently have removed through article 4 directions that we don't have data for on our [permitted development right dataset](https://www.planning.data.gov.uk/dataset/permitted-development-right), you can provide the legislation outlining the missing right and we could look to get this added to the dataset. Alternatively, emailing [digitalland@communities.gov.uk](https://www.planning.data.gov.uk/guidance/specifications/digitalland@communities.gov.uk) would also bring it to our attention.

Thanks again!

**Comment by kieranwint** [comment]
## Issue - Handling historic permitted development rights

We recently received a request from a local authority for some historic permitted development rights to be added to the dataset for the purposes of producing more accurate data for their article 4 direction dataset. 

Within the process of assessing the historic PDRs, we noticed some potential differences from our current model. The first is relating to our use of `part` and `class`. For example, in The Town and Country Planning (General Permitted Development)(England) Order 2015, [Part 1](https://www.legislation.gov.uk/uksi/2015/596/schedule/2/part/1/made) is related to "Development within the curtilage of a dwellinghouse", and [Class A](https://www.legislation.gov.uk/uksi/2015/596/schedule/2/part/1/crossheading/class-a-enlargement-improvement-or-other-alteration-of-a-dwellinghouse/made) is regarding "enlargement, improvement or other alteration of a dwellinghouse", **which is nested within Part 1**.

In historic legislation, for example the [1973 Town and Country Planning General Development Order](https://www.legislation.gov.uk/uksi/1973/31/pdfs/uksi_19730031_en.pdf), the first level after the schedule is `Class`, eg. Class V (being class 5), followed by sub-sections:
- 1;
- a;
- (i);
hence it is the reverse of what we currently have in the dataset with a modification to notation (roman numerals).

 We think this might work with our model, but we need to test to minimise unforeseen circumstances, and this is scheduled for the New Year.

As for further questions we have, is there still a restriction even though a PDR might be old or have been superseded? Are permitted development rights directly replaced when new legislation comes in? If permitted development rights are not directly replaced when new legislation is bought in, does this make the historic PDRs more enforceable?
Any thoughts on how PDRs are used in practice would be useful for us to know.
Thanks in advance!

**Comment by kieranwint** [comment]
## Historic permitted development rights and article 4 directions

We’ve recently had some queries about historic permitted development rights, and specifically how they link to [article 4 directions](https://github.com/digital-land/data-standards-backlog/discussions/30). As a result, we’ve done a bit of research to understand how things have evolved in regards to the legislation, with the aim of being able to trace back through this history for individual permitted development rights. The major pieces of legislation and high level information we’ve found are as follows:<br><br>
Legislation name | Amendments | Revoked in
-- | -- | --
The Town and Country Planning Act 1948 | 4 | 1959
The Town and Country Planning General Development Order 1959 | 2 | 1963
The Town and Country Planning General Development Order 1963 | 5 | 1973
The Town and Country Planning General Development Order 1973 | 3 | 1977
The Town and Country Planning General Development Order 1977 | 9 | 1988
The Town and Country Planning General Development Order 1988 | 16 | 1995
The Town and Country Planning (General Permitted Development) Order 1995 | 22 | 2015
The Town and Country Planning (General Permitted Development) (England) Order 2015 | 31 | n/a

For a more detailed version, please see the attached timeline we have produced. <br>
![Permitted Development Right legislation history_2025-04-17_11-24-23](https://github.com/user-attachments/assets/c551f8a4-61e5-4076-8d3e-7818e55d2bba)

**Reply by kieranwint** [reply]
Furthermore to the above post, we thought it might be a useful exercise to take an example permitted development right (in this case, 1E) and trace it back through legislation to its inception to show how it’s changed over time. Please see below:<br><br>
Link | Name | Description | Start date | End date
-- | -- | -- | -- | --
<a href="https://www.legislation.gov.uk/uksi/1948/958/pdfs/uksi_19480958_en.pdf">1948 legislation</a> | Development within the curtilage of a dwellinghouse | The erection, construction or placing, and the maintenance, improvement or other alteration, within the curtilage of any dwellinghouse, of:(i) any building or work (other than a dwelling or a garage) ancilliary to, but not forming part of, or permanently attached to, a dwellinghouse, and required for any purpose incidental to the enjoyment of the dwellinghouse as such, and;(ii) shelters for keeping poultry, livestock, bees, pet animals or birds for the domestic needs or personal enjoyment of the occupants of the dwellinghouse; | 1948-07-01 | 1963-04-30
<a href="https://www.legislation.gov.uk/uksi/1963/709/pdfs/uksi_19630709_en.pdf">1963 legislation</a> | Development within the curtilage of a dwellinghouse | The erection, construction or placing, and the maintenance, improvement or other alteration, within the curtilage of a dwellinghouse, of any building or enclosure (other than a dwelling, garage, stable, loose-box or coach-house) required for a purpose incidental to the enjoyment of the dwellinghouse as such, including the keeping of poultry, bees, pet animals, birds or other livestock for the domestic needs or personal enjoyment of the occupants of the dwellinghouse. | 1963-05-01 | 1973-02-28
<a href="https://www.legislation.gov.uk/uksi/1973/31/pdfs/uksi_19730031_en.pdf">1973 legislation</a> | Development within the curtilage of a dwellinghouse | The erection, construction or placing, and the maintenance, improvement or other alteration, within the curtilage of a dwellinghouse, of any building or enclosure (other than a dwelling, garage, stable, loose-box or coach-house) required for a purpose incidental to the enjoyment of the dwellinghouse, as such including the keeping of poultry, bees, pet animals, birds or other livestock for the domestic needs or personal enjoyment of the occupants of the dwellinghouse, so long as:(a)no part of such building or enclosure projects beyond the forewardmost part of any wall of the original dwellinghouse which fronts on a highway;(b)the height does not exceed, in the case of a building with a ridged roof, 4 metres, or in any other case, 3 metres;(c)the area of ground covered by buildings within the curtilage (other than the original dwellinghouse) does not thereby exceed one half of the total area of the curtilage excluding the ground area of the original dwellinghouse. | 1973-03-01 | 1977-03-28
<a href="https://www.legislation.gov.uk/uksi/1977/289/pdfs/uksi_19770289_en.pdf">1977 legislation</a> | Development within the curtilage of a dwellinghouse | The erection, construction or placing, and the maintenance, improvement or other alteration, within the curtilage of a dwellinghouse, of any building or enclosure (other than a dwelling, garage, stable, loose-box or coach-house) required for a purpose incidental to the enjoyment of the dwellinghouse, as such including the keeping of poultry, bees, pet animals, birds or other livestock for the domestic needs or personal enjoyment of the occupants of the dwellinghouse, so long as:(a)no part of such building or enclosure projects beyond the forewardmost part of any wall of the original dwellinghouse which fronts on a highway;(b)the height does not exceed, in the case of a building with a ridged roof, 4 metres, or in any other case, 3 metres;(c)the area of ground covered by buildings within the curtilage (other than the original dwellinghouse) does not thereby exceed one half of the total area of the curtilage excluding the ground area of the original dwellinghouse. | 1977-03-29 | 1981-03-31
<a href="https://www.legislation.gov.uk/uksi/1981/245/pdfs/uksi_19810245_en.pdf">1981 legislation</a> | Development within the curtilage of a dwellinghouse | The erection, construction or placing, and the maintenance, improvement or other alteration, within the curtilage of a dwellinghouse, of any building or enclosure (other than a dwelling, stable or loose-box) required for a purpose incidental to the enjoyment of the dwellinghouse as such, including the keeping of poultry, bees, pet animals, birds or other livestock for the domestic needs or personal enjoyment of the occupants of the dwellinghouse, so long as:(a)no part of such building or enclosure projects beyond the forewardmost part of any wall of the original dwellinghouse which fronts on a highway;(b)in the case of a garage or coachhouse, no part of the building is within five metres from any part of the dwellinghouse;(c)the height does not exceed, in the case of a building with a ridged roof, four metres, or in any other case, three metres;(c)the area of ground covered by buildings within the curtilage (other than the original dwellinghouse) does not thereby exceed fifty per cent of the total area of the curtilage excluding the ground area of the original dwellinghouse. | 1981-04-01 | 1988-12-04
<a href="https://www.legislation.gov.uk/uksi/1988/1813/schedule/2/part/1/chapter/5/made">1988 legislation</a> | Class E | The provision within the curtilage of a dwellinghouse of any building or enclosure, swimming or other pool required for a purpose incidental to the enjoyment of the dwellinghouse, or the maintenance, improvement or other alteration of such a building or enclosure. | 1988-12-05 | 1995-06-02
<a href="https://www.legislation.gov.uk/uksi/1995/418/schedule/2/part/1/crossheading/class-e">1995 legislation</a> | Class E | The provision within the curtilage of a dwellinghouse of any building or enclosure, swimming or other pool required for a purpose incidental to the enjoyment of the dwellinghouse as such, or the maintenance, improvement or other alteration of such a building or enclosure. | 1995-06-03 | 2015-04-14
<a href="https://www.legislation.gov.uk/uksi/2015/596/schedule/2/part/1/crossheading/class-e-buildings-etc-incidental-to-the-enjoyment-of-a-dwellinghouse">2015 legislation</a> | Buildings etc incidental to the enjoyment of a dwellinghouse | The provision within the curtilage of the dwellinghouse of—(a)any building or enclosure, swimming or other pool required for a purpose incidental to the enjoyment of the dwellinghouse as such, or the maintenance, improvement or other alteration of such a building or enclosure; or(b)a container used for domestic heating purposes for the storage of oil or liquid petroleum gas. | 2015-04-15 | n/a

<br>If there is anything here you believe to be incorrect, do let us know!<br>

**Comment by kieranwint** [comment]
Continuing from the previous message, we believe it is necessary to examine further instances of permitted development rights where the Article 4 direction that removes them refers to historic PDR legislation, and trace these through to the specific pieces of legislation. If you have any of these instances, please let us know.

One question we do have, and on which your input would be brilliant, is whether an Article 4 direction that was created to remove a historic PDR inherits any new legislative elements when the PDR legislation is subsequently amended. We recently consulted with the PDR policy team, who suggested that it depends on the wording of the direction, so we would appreciate your insights on this matter. Examples of such cases would also be greatly appreciated!

**Reply by trodaway** [reply]
One of our Legal Officers has offered the following advice:

> as to what happens to PD rights when a new GPDO is made, then (as far as I am aware) I understand that any existing Article 4 Direction made and confirmed will simply remain in force, unless revoked or replaced by a new Direction.
The PD rights under any particular GPDO are whatever was in force at the time under each respective GPDO.  An Article 4 Direction may simply remove or restrict one or more of any such rights from a particular site or area, so that planning consent is then needed.  As such, you would always need to consider whether any Article 4 Direction in place affects any of the PD rights that would normally apply.

They've also suggested that our Head of Planning probably knows far more about this area of law than they do and may be able to offer a different perspective - once I hear this, I'll share it too.

**Reply by kieranwint** [reply]
Thanks for this response @trodaway. This is a good first indicator, and thanks in advance for anything you can gather from your Head of Planning!
We have also emailed the legal team from our side with an example to hopefully provide them with some clarity on the queries at hand, and will of course update this thread when we receive a response and can begin taking action.

**Reply by trodaway** [reply]
@kieranwint, our Head of Planning and others in our Planning team have discussed this amongst themselves and the conclusion I have been provided with is that they agree with the advice from our Legal Officer and that they have nothing further to add. Hopefully you've managed to get some feedback from your legal team.

**Comment by psd** [comment]
@Swati-Dash reports that there may be article 4 directions which remove "all permitted development rights". We could support this with a `permitted-development-right` of `all` or some such.

Conversely, there may be cases where the permitted development right is removed in part. Here we can add a code for each sub-section with the right in the parent field. 

I also wonder if modelling the permitted-development-right-part as a separate dataset, rather than treating this as a hierarchical code with a permitted-development-right-parent field given we hear that there are directions which remove all of the rights under a part, or all of the rights (the root of the hierarchy).

**Reply by colmjude** [reply]
That sounds like an interesting use case.

Do we have any examples of these?
Any that cover ALL permitted development rights?
Any that cover removing an entire part, for example, ALL the **1**s (1A, 1B, 1C, etc.)?

**Comment by drewhardy77** [comment]
Based on regular and ongoing feedback from several originating authorities and users of the data, including OSL/PlanX, we believe there may be a good case for allowing further granularity in classification beyond the current class/part divide. It is often the case that a given A4 direction will only apply to a subsection of a part, and the current recommendation is that this should be recorded in the notes section.

_Example_:
```
3L - Small HMOs to dwellinghouses and vice versa

Development consisting of a change of use of a building: 
(a) from a use falling within Class C4 (houses in multiple occupation) of the Schedule to the Use Classes Order, to a use falling within Class 3 (dwellinghouses) of the Schedule; 
(b) from a use falling within Class C3 (dwellinghouses) of the Schedule to the Use Classes Order, to a use falling within Class C4 (houses in multiple occupation) of that Schedule
```

In many cases, an Article 4 will only remove the PD right for sub-part (b) - changing a dwellinghouse into an HMO, rather than rights associated with (a). [This example from Brighton](https://www.brighton-hove.gov.uk/sites/default/files/2021-08/Article%204%20Direction.pdf) is typical 

**Why a change being considered?**

1. Amending the specification to allow users to specify which clauses apply in a standardised way allows for the automation of services e.g. PlanX, in a way that free text entries in the notes section does not
2. Providers of the data are asking for clarity on this on a regular basis

**Suggested changes for discussion**

Option 1: Amend guidance to make it clear that applicable sub-parts should be referred to in the notes section (continuation/formalisation of current practice)
Option 2: Create additional optional field for sub-part (or whatever the agreed term is for the clause) in which ONLY the relevant sub-parts are recorded, separated by a semi-colon if multiple are required. The `reference` value would remain the same and would be recorded in the same way by originating authorities e.g. the PDR referenced above would still be recorded as 3L, rather than 3Lb. The only difference would be the addition of a new attribute (name to be agreed) with the sub-part reference in it e.g. `permitted-development-right-sub-part:b`
Option 3: Same as Option 2, but new `reference` values are created for each variation. Knock on effects on the dataset size and complexity may be unwelcome.
Option 4: Do nothing and continue with current practice

Option 2 would be my preferred approach for clarity. It would be useful to agree on terminology for referring to these sub-parts/clauses. `permitted-development-right-clause`, `permitted-development-right-sub-part`, or something else?
